src/app/technical-record/test-history/test-history.component.html

```diff
@@ -55,7 +55,7 @@
         </td>
         <td id="{{ 'test-view-' + i }}" class="govuk-table__cell">
           <a
-            class="govuk-link"
+            class="govuk-link link--blue"
```

Comment:
try this `<a
                    href="javascript:void(0)"
                    class="govuk-link"`